canonical-websites/www.ubuntu.com
# 1704 takeover

## Done

Build new 17.04 takeover
Replace release end of life image 

## QA

- Pull code and run ./run
- Go to http://localhost:8001/
- Check the page in s/m and l, check the code and check against copy doc
- Check that the eol image on /desktop/enterprise is this one https://assets.ubuntu.com/v1/cf53cf6c-release-ubuntu-end-of-life.png
- Demo available at www.ubuntu.com-1704-takeover.demo.haus

## Issue / Card

Card: https://trello.com/c/RIMP9PH5
Doc: https://docs.google.com/document/d/1q9w5WklFn5jA5a3kuLHWXO2iLaQopoponvIx2pYoLrg/edit
Design: https://www.dropbox.com/home/00_web_team/_ubuntu.com/takeovers/17.04
